libcudacxx/include/cuda/cmath

```diff
@@ -31,7 +32,8 @@ _LIBCUDACXX_BEGIN_NAMESPACE_CUDA
 template <class _Tp, _CUDA_VSTD::__enable_if_t<_CCCL_TRAIT(_CUDA_VSTD::is_integral, _Tp), int> = 0>
 _CCCL_NODISCARD _LIBCUDACXX_HIDE_FROM_ABI _CCCL_CONSTEXPR_CXX14 _Tp ceil_div(const _Tp __a, const _Tp __b) noexcept
 {
-  _LIBCUDACXX_DEBUG_ASSERT(__a >= _Tp(0), "cuda::ceil_div: a must be non negative");
+  _LIBCUDACXX_DEBUG_ASSERT(_CCCL_TRAIT(_CUDA_VSTD::is_unsigned, _Tp) || __a >= _Tp(0),
+                           "cuda::ceil_div: a must be non negative");
```

Comment:
how is this change related to the `__host__`/`__device__` annotations?

Reply:
That is most likely a leftover from a different PR that sneaked in